docs/roadmap.md

```diff
@@ -4,25 +4,27 @@
 
 **yunikorn-core**
 
-* Gang Scheduling.
-* Priority based preemption.
-* Support application priority.
-* Rich scheduler metrics.
-* Workload simulator.
-* Prometheus integration (phase 2).
-* Grafana integration.
+* [YUNIKORN-1](https://issues.apache.org/jira/browse/YUNIKORN-1): Support app/task priority aware scheduling.
+* [YUNIKORN-2](https://issues.apache.org/jira/browse/YUNIKORN-2): Gang Scheduling.
+* [YUNIKORN-21](https://issues.apache.org/jira/browse/YUNIKORN-21): Optimize node sorting algorithms.
+* [YUNIKORN-42](https://issues.apache.org/jira/browse/YUNIKORN-42): High efficient scheduling events framework phase 1.
+* [YUNIKORN-33](https://issues.apache.org/jira/browse/YUNIKORN-33): Performance benchmark with Kubemark.
+* [YUNIKORN-131](https://issues.apache.org/jira/browse/YUNIKORN-131): Prometheus integration - phase 2.
+* [YUNIKORN-132](https://issues.apache.org/jira/browse/YUNIKORN-132): Publish helm charts for convenient access.
 
 **yunikorn-k8shim**
 
-* Security support.
-* Performance improvement: merging, packing requests.
-* 3rd Operator/controller integration, pluggable app discovery.
```

Comment:
How about detailed APP state management including FINISHED & FINISHING and other closing states?

Reply:
That is more like a bug fix

Reply:
Basic APP state is there anyway, I am fine with this.